### PR TITLE
minor cleanup

### PR DESCRIPTION
**Pull request by yogthos**

### CODE REVIEW

File: src/to_jdbc_uri/core.clj [hunk 1]
```diff
@@ -1,19 +1,29 @@
-;;;; Copyright © 2015 Carousel Apps, Ltd
+(ns to-jdbc-uri.core
```
Comment (pupeno) [inline src/to_jdbc_uri/core.clj:3]: Can you please re-add the copyright notice?

Comment (yogthos) [inline src/to_jdbc_uri/core.clj:3]: oh oops totally missed that ;)

**Comment by pupeno** [comment]
@yogthos do you need a release with this?

**Comment by yogthos** [comment]
I don't think so, the functionality is still the same.

**Pull request merged**